README.md

```diff
@@ -33,29 +33,29 @@ Run `make run` to start the operator.
 - Get HTTP server address by running: `HTTP_SERVER=$(oc get routes flotta-operator-controller-manager -n flotta --no-headers -o=custom-columns=HOST:.spec.host)`.
 - Start yggdrasil with from the yggdrasil repository directory: 
    ```
-   sudo go run ./cmd/yggd \
-     --log-level info \
-     --protocol http \
-     --path-prefix api/flotta-management/v1 \
-     --client-id $(cat /etc/machine-id) \
-     --cert-file /etc/pki/consumer/cert.pem \
-     --key-file /etc/pki/consumer/key.pem \
-     --server $HTTP_SERVER
+    sudo ./yggd \
+    --log-level info \
+    --protocol http \
+    --path-prefix api/flotta-management/v1 \
+    --client-id $(cat /etc/machine-id) \
+    --cert-file /etc/pki/consumer/cert.pem \
+    --key-file /etc/pki/consumer/key.pem \
+    --server $HTTP_SERVER
    ```
 
 ### On minikube
 - Deploy the operator: `IMG=<image registry and tag> HOST=<host name> TARGET=k8s make deploy`.
 - Add to /etc/hosts: `<minikube ip> <host name>`.
 - Start yggdrasil with from the yggdrasil repository directory:
    ```
-   sudo go run ./cmd/yggd \
-     --log-level info \
-     --protocol http \
-     --path-prefix api/flotta-management/v1 \
-     --client-id $(cat /etc/machine-id) \
-     --cert-file /etc/pki/consumer/cert.pem \
-     --key-file /etc/pki/consumer/key.pem \
-     --server <host name>
+    sudo ./yggd \
+    --log-level info \
+    --protocol http \
+    --path-prefix api/flotta-management/v1 \
+    --client-id $(cat /etc/machine-id) \
+    --cert-file /etc/pki/consumer/cert.pem \
+    --key-file /etc/pki/consumer/key.pem \
+    --server $HOST
```

Comment:
on minikube it supposed to be the name of the ingress, no? 
I mean one should use `flotta-operator.srv` or the host variable provided to Makefile at build.
And add it into the local /etc/hosts

Reply:
I wasn't able to start minikube by setting the `<minikube ip> <host name>` in `/etc/hosts`.
It works only using `--server localhost:8888` (in my case). 
Maybe something is wrong in this section?

Reply:
@masayag yes, it is the `HOST` which was provided during make deploy. If not, the default is set to `flotta-operator.srv`.
@gciavarrini Can you please elaborate? Did you install yggdrasil before starting it with the host name as mentioned [here](https://github.com/project-flotta/flotta-device-worker#start-yggdrasil)?

Reply:
Here the steps (on my local machine):

`minikube  delete`
`minikube start`
Add `minikube ip` to `/etc/hosts` (e.g. `192.168.39.110 operator-hostname`)

On `flotta-operator` I execute:
 `make install`
 `make run`

On the device I execute :
`sudo rm -f /usr/local/var/run/yggdrasil/workers/device-worker.pid`
`export GOPROXY=proxy.golang.org,direct`
`LIBEXECDIR=/usr/local/libexec make install`

Then if I start `yggdrasil` with 
```sudo ./yggd \                                                                                                                ✱
  --log-level debug \
  --protocol http \
  --path-prefix api/flotta-management/v1 \
  --client-id $(cat /etc/machine-id) \
  --cert-file /etc/pki/consumer/cert.pem \
  --key-file /etc/pki/consumer/key.pem \
  --socket-addr @yggd \
  --server operator-hostname
   ```
  
then I see the log messages
` cannot send connection status message: cannot post to URL: Post "http://operator-hostname/api/flotta-management/v1/control/d63bfb403c82493f8c1a780c8b78c421/out": dial tcp 192.168.39.110:80: connect: connection refused
`
If I use ` --server localhost:8888` everything works fine.

@arielireni what do you think? What I'm doing wrong?

Reply:
Can you please try to change the line in `/etc/hosts` to `192.168.39.110 flotta-operator.srv` and start yggdrasil with: 
 ``` --log-level debug \
  --protocol http \
  --path-prefix api/flotta-management/v1 \
  --client-id $(cat /etc/machine-id) \
  --cert-file /etc/pki/consumer/cert.pem \
  --key-file /etc/pki/consumer/key.pem \
  --socket-addr @yggd \
  --server flotta-operator.srv```
  instead?
  I think that since you didn't deploy the flotta-operator with `IMG=<image registry and tag> HOST=<host name> TARGET=k8s make deploy` the default HOST is `flotta-operator.srv` and not `operator-hostname` as you defined